src/tiles/grid/item-tile.ts

```diff
@@ -35,6 +35,10 @@ export class ItemTile extends LitElement {
 
   render() {
     const itemTitle = this.model?.title;
+    const [viewCount, viewLabel] =
+      this.sortParam?.field === 'week'
+        ? [this.model?.weeklyViewCount, 'weekly views']
+        : [this.model?.viewCount, 'all-time-views'];
```

Comment:
QUESTION: Inconsistent value formatting? Should there be a space before the word "views" in both cases?

Reply:
Ah, good catch! Typo should be fixed in the latest commit.